testdata/print-non-module.txt

```diff
@@ -1,12 +0,0 @@
-env HOME=$WORK/home
-! gobin -p github.com/gobin-testrepos/non-module
```

Comment:
why not add a test that checks that this command now succeeds as expected?

Reply:
Good point 👍